packages/flutter_tools/lib/src/features.dart

```diff
@@ -2,19 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
-
-import 'package:meta/meta.dart';
-
-import 'base/config.dart';
 import 'base/context.dart';
-import 'base/platform.dart';
-import 'version.dart';
 
 /// The current [FeatureFlags] implementation.
 ///
 /// If not injected, a default implementation is provided.
```

Comment:
> If not injected, a default implementation is provided.

That's not true, right? I think you may hit a null error in context_test:

https://source.corp.google.com/piper///depot/google3/mobile/flutter/cli/lib/context.dart;l=673?q=%20FeatureFlags&ss=piper%2FGoogle%2FPiper:google3%2Fmobile%2Fflutter%2Fcli%2F

Similar to the change I made in:
https://critique-ng.corp.google.com/cl/365154959/depot/google3/mobile/flutter/cli/test/context_test.dart

You may need to add this to `context_test` `Honors a specified project platform`:
```dart
FeatureFlags: () => Google3Features(),
```
And make `Google3Features` visible for testing.

Reply:
Ahh yeah, I think this used to have a default...

Reply:
you were correct :D

Reply:
🔮